examples/protoschool.md

```diff
@@ -0,0 +1,7 @@
+# IPFS Tutorials at ProtoSchool
+
+![image](https://user-images.githubusercontent.com/19171465/87570374-8fe89580-c696-11ea-8cf5-acc29d9761df.png)
+
+Explore [ProtoSchool's IPFS tutorials](https://proto.school/#/tutorials?course=ipfs) for interactive js-ipfs coding challenges, deep dives into DWeb concepts like content addressing, and more. 
+
+![image](https://user-images.githubusercontent.com/19171465/87570061-0769f500-c696-11ea-8b94-08d2de8e2139.png)
```

Comment:
This image is a picture of the protoschool website - it's got things that look clickable - a drop down, checkbox, etc.

Could it be something else?  A description of the courses or something?  Or failing that, include the browser chrome in the picture so the user knows straight away they are looking at a picture of a website rather than something can interact with.

Also it should probably be a link to the protoschool website.